packages/x-components/src/components/layouts/multi-column-max-width-layout.vue

```diff
@@ -69,14 +69,16 @@
         </Scroll>
       </BaseIdTogglePanel>
 
-      <Scroll v-if="hasContent('main-body')" id="main-scroll" class="x-layout__body-scroll">
-        <section class="x-layout__main-body x-list x-list--vertical">
-          <!-- @slot Slot that can be used to insert the body content. -->
-          <slot name="main-body">
-            <span v-if="devMode" class="slot-helper" style="height: 110vh">MAIN BODY</span>
-          </slot>
-        </section>
-      </Scroll>
+      <MainScroll>
```

Comment:
The other layout components are jealous, they deserve a `MainScroll` too

Reply:
I was asking in the other PR if we should treat the scroll as a special case and allow it to be used inside layouts. This makes that the `scroll` module is always present, even if you don't use our layouts or a scroll component, until the tree-shaking step happens and removes the unused code, which will only happen in production builds. 

To be honest I don't like this, I would like to fix this in another task. These are some options that came to my mind:

- Leave the layouts without the configured scroll.
- Move the layouts to its own independent module that would depend on the scroll module.
- Make the current HTMLElements in the layout configurable, so you can pass the component you want to render using a prop.

Reply:
We leave changed at the moment, and we can refactor later in other PR, but we need the current layouts working with URL scroll